exercises/practice/resistor-color/src/lib.rs

```diff
@@ -0,0 +1,26 @@
+#[derive(Debug, PartialEq)]
+pub enum ResistorColor {
+    White,
+    Grey,
+    Violet,
+    Blue,
+    Green,
+    Yellow,
+    Orange,
+    Red,
+    Brown,
+    Black,
```

Comment:
Why is this sorted backwards? It should be sorted either forwards, or alphabetically.

Reply:
I think alphabetically makes sense, so that the student has to read the instructions in order to make correct enum.